packages/core/src/helpers/border-radius/border-radius.scss

```diff
@@ -1,10 +1,9 @@
+@use '@onfido/castor-tokens' as tokens;
```

Comment:
This is an odd one.

Basically, running with `sass-cli` does not find `node_modules` imports if using:

```scss
@use '~@onfido/castor-tokens';
```

Looks like a solution is to use [load paths](https://sass-lang.com/documentation/at-rules/use#load-paths), however then it should not have `~`.

But then using a helper in app fails, e.g. here: https://github.com/onfido/castor/pull/662/checks?check_run_id=2622081236

Not sure yet what to do here TBH.

Reply:
afaik that's Parcel 1 that requires the starting `~`, Parcel 2 does not.
Don't remember what Webpack does.

This will be tricky 😬

Reply:
From our docs, Webpack seems to require the tilde `~` too 🙈

Reply:
I've refactored all that part, now I'm integrated the whole `src` upon `yarn install`. So Castor Tokens becomes part of Core source.